package.json

```diff
@@ -12,6 +12,7 @@
     "@emotion/cache": "^11.11.0",
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
+    "@mui/icons-material": "^5.15.20",
```

Comment:
We decided to use bootstrap icons because they're a bit more generic and we use them in other products. They can be used in a similar way, [here's an example](https://github.com/kausaltech/nzc-data-studio/blob/main/components/AppBar.tsx#L91)

https://icons.getbootstrap.com/